src/models/message.ts

```diff
@@ -396,4 +396,66 @@ export class MessageModel extends DocumentDbModel<
     const content = contentOrError.value;
     return right<Error, Option<MessageContent>>(some(content));
   }
+
+  public async storeContentAsBlob(
```

Comment:
why we keep `attachStoredContent` / `gettStoredContent` ?

Reply:
Removed